src/sonar-scanner-executable.js

```diff
@@ -100,9 +101,16 @@ function getSonarScannerExecutable(passExecutableCallback) {
   const baseUrl = process.env.SONAR_SCANNER_MIRROR || process.env.npm_config_sonar_scanner_mirror || SONAR_SCANNER_MIRROR
   const fileName = 'sonar-scanner-cli-' + platformBinariesVersion + '-' + targetOS + '.zip'
   const downloadUrl = baseUrl + fileName
+  const proxy = process.env.http_proxy || ''
+  // create an instance of the `HttpsProxyAgent` class with the proxy server information
+  const proxyAgent = new HttpsProxyAgent(proxy)
```

Comment:
Not sure we should be creating the instance if there is no proxy value

Reply:
I did some changes for the log and safely moved the instance creation inside the condition